src/sentry/static/sentry/app/views/settings/components/forms/controls/radioBoolean.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+
+interface OnChangeHandler {
```

Comment:
Why an interface instead of type alias?

Reply:
No real reason. I'm happy to go with a type alias as we have more of those.

Reply:
Yeah let's do it, I feel like we rarely use interfaces